qdrant_client/qdrant_client.py

```diff
@@ -23,15 +23,14 @@ class QdrantClient(QdrantFastembedMixin):
         This module methods are wrappers around generated client code for gRPC and REST methods.
         If you need lower-level access to generated clients, use following properties:
 
-        - :meth:`QdrantClient.grpc_points`
-        - :meth:`QdrantClient.grpc_collections`
-        - :meth:`QdrantClient.rest`
+        - :py:attr:`QdrantClient.grpc_points`
```

Comment:
I am wondering if QdrantClient can be replaced with some reference to the class itself? (not sure if it is possible in sphinx)

I am worried because AsyncQdrantClient has links to the properties of QdrantClient :( (and I started thinking about the ways to handle it)